blocks/quote/quote.css

```diff
@@ -0,0 +1,29 @@
+.quote blockquote {
+  margin: 0;
```

Comment:
I would change the style
```css
    max-width: 60%;
    margin: 0 auto;
    font-style: italic;
```
and for `quote-attribution`:
```css
text-align: right;
```
![Screenshot 2024-01-10 at 09 49 25](https://github.com/adobe/helix-block-collection/assets/2250964/326d1fcb-1924-4e36-a9dd-7045bb72a141)